Marlin/src/gcode/gcode.cpp

```diff
@@ -701,8 +701,7 @@ void GcodeSuite::process_next_command() {
    */
   void GcodeSuite::process_subcommands_now_P(const char *pgcode) {
     // Save the parser state
-    char saved_cmd[strlen(parser.command_ptr) + 1];
-    strcpy(saved_cmd, parser.command_ptr);
+    char *saved_cmd = parser.command_ptr;
```

Comment:
Hmm, yeah. This should work ok. The command pointer points to a slot in the command queue which won't get overwritten until later on. The slot is blocked from being re-used until the command exits.